php/stats.php

```diff
@@ -39,101 +41,121 @@
     $sth->execute([$user]);
     $row = $sth->fetch();
     if (!$row) {
-        die('Error;' . _("User not found."));
+        die(json_encode(["error" => ("User not found.")]));
     }
 
     $public = $row["public"];
     if ($public == "N") {
-        die('Error;' . _("This user's flights are not public."));
+        die(json_encode(["error" => ("This user's flights are not public.")]));
```

Comment:
Localistion dropped? `_(` has become `(`; brackets aren't adding anything in the current form

Could potentially use `json_error()` from `helper.php`.